packages/orbit-components/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -78,108 +69,77 @@ export const WithTextLinkInLabel = ({ checked }) => {
           .
         </Text>
       }
-      checked={checked}
-      value="value"
-      onChange={action("changed")}
     />
-  );
-};
-
-WithTextLinkInLabel.story = {
-  name: "With TextLink in label",
+  ),
 
   parameters: {
     info: "Additionally you can add info to this component.",
+    controls: {
+      exclude: ["label"],
+    },
   },
-};
 
-WithTextLinkInLabel.args = {
-  checked: true,
+  args: {
+    checked: true,
+  },
 };
 
-export const WithTooltip = () => {
-  return (
-    <Checkbox
-      label="Direct"
-      value="one"
-      disabled
-      tooltip={
-        <Tooltip content="There are no results available with this option" placement="top" />
-      }
-    />
-  );
-};
+export const WithTooltip: Story = {
```

Comment:
When I checked this one with tooltip, the ✔️ is not there, do we have a bug here? 🙀